### ISSUE TITLE
Select edge inside a node-group

### ISSUE THREAD

**Issue by rcambeirooreiro** [artem-mangilev/ngx-vflow#101]
In the current version, it is not possible to select and delete edges that are inside a group node. I have read that this might be related to how SVG elements are rendered, with nodes being drawn first and then edges on top, similar to how z-index works in HTML.

Is there a way to ensure that edges inside group nodes can be selected and deleted properly? Any suggestions on how to resolve this, possibly by adjusting the SVG rendering order?

**Comment by artem-mangilev** [comment]
Thanks for the issue ! I'll research how to fix this during v0.16

**Comment by artem-mangilev** [comment]
You’re correct about the rendering order issue. The problem was that the edges were positioned behind the nodes, causing the browser not to trigger click events on edges that were overlapped by group nodes. 

As a quick solution, we could separate nodes into two categories: group nodes and regular nodes. Group nodes would be rendered behind the edges, while regular nodes would remain in front. This approach resolves your case but introduces a minor drawback—group nodes can no longer overlap regular nodes due to the new layering order. 

I would like to avoid changing behavior to everyone so I’ll include a flag in the next release, allowing you to toggle between the old layering and the new one with the old layering as a default.

**Comment by artem-mangilev** [comment]
@rcambeirooreiro Hi! Could you please try to enable `detachedGroupsLayer` flag in this input of `<vflow />`

<img width="860" alt="image" src="https://github.com/user-attachments/assets/f271e75c-791c-42e7-a75e-c3230b6aae72">
